numbersofinterest/OptionsTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace NumbersOfInterest;
+require_once __DIR__ . '/Options.php';
+use NumbersOfInterest\Options;
+
+class OptionsTest extends \PHPUnit_Framework_TestCase
```

Comment:
nice to test this in isolation, also with the corner cases such as invalid input